.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ workflows:
 executors:
   build:
     docker:
-      - image : prestocpp/velox-circleci:mikesh-20211012
+      - image : prestocpp/velox-circleci:sagarm-20220131
```

Comment:
Thanks Sagar, just want to reiterate that we probably need some pipeline to produce these images rather than devs. I think that work is on my plate this half.

Reply:
Yes, please! I look forward to losing my docker push permissions :)